src/controllers/profile/updateProfile.js

```diff
@@ -0,0 +1,60 @@
+import User from '../../models/user';
+
+export default {
+  profileImage: async (req, res) => {
+    try {
+      if (!req.file) return res.status(400).send({ msg: 'your photo can\'t be saved' });
+
+      const user = await User.findByIdAndUpdate(req.user._id,
+        { photoUrl: req.file.path },
+        { new: true });
+
+      return res
+        .status(200)
+        .send({
+          msg: 'Profile Picture updated successfully',
+          user: {
+            firstName: user.firstName,
+            secondName: user.secondName,
+            username: user.username,
+            phoneNumber: user.phoneNumber,
+            email: user.email,
```

Comment:
Hey @Habinezajanvier usually, we don't allow users to add an email while creating/updating a profile. We normally update an email through settings page, which has some added security layer to prevent anyone from updating the profile which is not theirs. Therefore, it is a good idea to remove email from the profile section. Normally, when a user creates an account, we take it as if he has created a profile but with email only, later they will come to update their profile. Which means we don't need to have create profile in the first place. Not sure if it makes sense but we can talk about it later.